mobile/components/FormCard.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useState } from "react";
+import { View, styled } from "tamagui";
+import Badge, { BadgeProps } from "./Badge";
+import Card from "./Card";
+import { Typography } from "./Typography";
+import CardFooter from "./CardFooter";
+
+export interface FormCardProps {
+  /**
+   * Header text
+   */
+  header: string;
```

Comment:
i think i would rename this headerText, as i would expect an entire component just for header instead of just the text that goes into it